### PR TITLE
Remove explicit trailing underscore in analysis_key in Multiple coding mode

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Copied from https://github.com/AfricasVoices/Project-JPLG-2020/pull/33/files

### CODE REVIEW

File: automated_analysis.py [hunk 1]
```diff
@@ -601,7 +601,7 @@ def set_survey_percentages(survey_counts, total_survey_counts):
                     "Gender": gender_code.string_value,
                     "Number of Participants": demographic_counts[f"gender:{gender_code.string_value}"],
                     "Fraction of Relevant Participants": None if total_relevant_gender == 0 else
-                        demographic_counts[f"gender:{gender_code.string_value}"] / total_relevant_gender
+                    demographic_counts[f"gender:{gender_code.string_value}"] / total_relevant_gender
```
Comment (as2388) [inline automated_analysis.py]: I think this was easier to read with the indent

**Review by as2388 — APPROVED** [review]
Please confirm you've done an end-to-end test, as it's a bit hard to find all the cases that need to change, especially from a code review

File: automated_analysis.py (hunk 1 above)
Comment (IsaackMwenda) [reply automated_analysis.py]: updated thanks.

**Comment by IsaackMwenda** [comment]
> Please confirm you've done an end-to-end test, as it's a bit hard to find all the cases that need to change, especially from a code review

Yes, compared analysis files and they match.

**Pull request merged**